src/kernl/implementations/attention_skinny.py

```diff
@@ -88,48 +88,48 @@ def _fwd_part_1(
     maximums_head_stride,
     maximums_step_stride,
     maximums_m_stride,
-    o_batch_stride,
-    o_head_stride,
-    o_step_stride,
-    o_m_stride,
-    o_n_stride,
+    output_batch_stride,
+    output_head_stride,
+    output_step_stride,
+    output_m_stride,
+    output_n_stride,
     attention_mask_batch_stride,
     attention_mask_head_stride,
     attention_mask_m_stride,
     attention_mask_k_stride,
     min_clamp_value,
-    NEED_LOAD_MASK_SIZE_N: tl.constexpr,
-    NEED_LOAD_MASK_SIZE_M: tl.constexpr,
+    N_LOAD_MASK_NEEDED: tl.constexpr,
+    M_LOAD_MASK_NEEDED: tl.constexpr,
     MASK_BATCH_SIZE: tl.constexpr,
     MASK_HEAD_SIZE: tl.constexpr,
     MASK_M_SIZE: tl.constexpr,
     MASK_K_SIZE: tl.constexpr,
     HAS_MASK: tl.constexpr,
     IS_CAUSAL: tl.constexpr,
-    BLOCK_M: tl.constexpr,
-    BLOCK_DHEAD: tl.constexpr,
-    BLOCK_N: tl.constexpr,
+    BLOCK_M_SIZE: tl.constexpr,
+    BLOCK_DHEAD_SIZE: tl.constexpr,
+    BLOCK_N_SIZE: tl.constexpr,
 ):
     # Index of the block on M axis (M axis is the rows of matrix K)
     n_block_idx = tl.program_id(0)
```

Comment:
Should it be `block_n_idx` and `block_m_idx` ?

Reply:
done